aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,54 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
+  - Gain precise control over how page layouts respond and adapt.
+  - Implement grid layouts, such as a [dashboard of cards](/templates/dashboards#two-column-dashboard), which gracefully adapt and flow.
+  - Is your application currently consuming Grommet's ResponsiveContext? If so, the additional breakpoints are breaking changes from v2 and may require minor code changes to ensure backwards compatibility. A [migration guide](https://github.com/grommet/grommet-theme-hpe/wiki/Migrating-from-HPE-Theme-2.x-to-3.x) is available to make this a smooth process.
+
+## March 9, 2022 - April 1, 2022
+
+### Communications
+
+- To help scale and accelerate Design System feature delivery, we are actively working on a process for receiving design contributions from members outside of the Design System core team. Initially, designers will be able to leverage [Figma's Branching feature](https://www.figma.com/best-practices/branching-in-figma/) to contribute small fixes and medium enhancements while the process is tested and refined. Stay tuned!
+
+
+### Roadmap Updates
+
+- **Page Layouts**
+  - Updated the [Page Layouts Figma file](https://www.figma.com/file/4rdBkUlHd5MCVq3hvUOXHd/Page-Layouts?node-id=2891%3A21273) to reflect new Page component in Grommet. Page component guidance and examples coming soon to the Design System site.
+- **Global Banner Notifications**
+  - [Guidance and examples for Global Banner Notifications](/templates/global-banner-notifications) are now available.
+  - Styling is supplied by the HPE Theme and available on `grommet-theme-hpe v2.4.0`.
+- **Page Header**
+  - Began work to standardize [Page Headers] (https://www.figma.com/file/BHp0KrOcPvbnzNyshfHkd8/HPE-Page-Header?node-id=84%3A2940) across HPE applications. The scope of this work is to define how page titles, page level actions, and contextual information should be displayed.
+- **Tabs**
+  - Began working with stakeholders to understand uses cases and standardize scenarios with more than 5 Tabs across HPE applications. The scope of this work is to define how to visually display tabs across different resolutions.
+
+
+### Stay Current with the Latest Changes
+- Want to know how to create dynamic, responsive layouts? Gain a better understanding [Grid's capabilities and how to master this component](/components/grid#grid-properties-and-key-concepts) to create dynamic layouts.
+- Added an example of a Form with a [single required FormField](/templates/forms#single-required-formfield) on the site.
+- FileInput - It was found that the Trash icon was tied with something being unrecoverable which is not the case in removing a file in a FileInput. [remove file action](/components/fileinput) is now represented by the Close icon instead of the Trash icon. This change has been made in the Figma component and is available in the grommet-hpe-theme v2.4.0.
```

Comment:
```suggestion
- FileInput - The [remove file action](/components/fileinput) is now represented by the Close icon instead of the Trash icon. This aligns the icon more closely with the action it is representing (the Trash icon was associated with destroy/delete/unrecoverable, which is not the case in removing a file from a FileInput). This change has been made in the Figma component and is available in the grommet-hpe-theme v2.4.0.
```